src/components/cardlist.jsx

```diff
@@ -0,0 +1,28 @@
+// eslint-disable jsx-props-no-spreading
```

Comment:
Nope, that's not how you work with ESLint :p

Reply:
done